radius.go

```diff
@@ -22,22 +27,44 @@ func start_radius_server(db *sql.DB) {
 		// grabbing information
 		// note that we assume username = imsi
 		// optional ip is undefined
-		username := rfc2865.UserName_GetString(r.Packet)
-		// password := rfc2865.UserPassword_GetString(r.Packet)
-		var code radius.Code
-		_, err := db.Query("SELECT *  FROM customers WHERE customers.imsi = ?", username)
+		imsi := rfc2865.UserName_GetString(r.Packet)
+		// query through database to find if this imsi matches some IP
+		// if it does, write back the IP
+		// Reject otherwise
+		rows, err := db.Query("SELECT ip FROM static_ips WHERE imsi = ?", imsi)
+		defer rows.Close()
+		var res *radius.Packet
 		if err != nil {
+			// we should reject the user 
 			log.WithError(err).Error("Cannot find this customer in our db")
-			code = radius.CodeAccessReject
+			res = radius.New(radius.CodeAccessReject, []byte(`res`)) // what should we set the secret?
 		} else {
-			code = radius.CodeAccessAccept
-		}
+			// we should accept the user
+			res = radius.New(radius.CodeAccessAccept, []byte(`res`)) // what should we set the secret?
+			// grab the IP
+			var ip string
+			for rows.Next() {
+				if err1 := rows.Scan(&ip); err1 != nil {
+        			log.Fatal(err1)
```

Comment:
A fatal log message like this should probably have some more context. It will be pretty hard to debug if we hit it right now, since fatal will kill the program. Maybe consider also logging the ip as a parameter to the structured log?